src/coreclr/gc/gcevents.h

```diff
@@ -48,5 +48,11 @@ KNOWN_EVENT(PrvSetGCHandle, GCEventProvider_Private, GCEventLevel_Information, G
 KNOWN_EVENT(PrvDestroyGCHandle, GCEventProvider_Private, GCEventLevel_Information, GCEventKeyword_GCHandlePrivate)
 KNOWN_EVENT(PinPlugAtGCTime, GCEventProvider_Private, GCEventLevel_Verbose, GCEventKeyword_GCPrivate)
 
+// TODO, AndrewAu, naming
+// TODO, AndrewAu, level
+DYNAMIC_EVENT(CommittedUsage, GCEventLevel_Information, GCEventKeyword_GC)
+DYNAMIC_EVENT(DynamicHeapCountData, GCEventLevel_Verbose, GCEventKeyword_GC)
```

Comment:
perhaps call these 2 events something like 
`DynamicHeapCountSample" and `DynamicHeapCountTuning"? (or add Data at the end)